File: VertaskAPI/src/main/java/br/cefetmg/vertask/model/Empresa.java
package br.cefetmg.vertask.model;

import lombok.*;
import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Empresa {
    Long idEmpresa;
    String nome;
    String cnpj;
    String fantasia;
    String endereco;
    String telefone;
    String email;
    BigDecimal honorario;
}

File: Vertask - Ionic/src/app/model/empresa.ts
export class Empresa {
  idEmpresa: number = 0;
  nome: string = '';
  cnpj: string = '';
  fantasia: string = '';
  endereco: string = '';
  telefone: string = '';
  email: string = '';
  honorario: number = 0;

  constructor(init?: Partial<Empresa>) {
    Object.assign(this, init);
  }
}
